src/web/services/__snapshots__/student.service.spec.ts.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StudentService should generate course student list with section as csv 1`] = `
+"Course ID,studentListWithSection
+Course Name,Student List With Section
+
+Section,Team,Full Name,Last Name,Status,Email
```

Comment:
We shall have at least one student with the last name passed down.

Reply:
What do you mean?

Reply:
You did not provide `lastName` (while this is not true in production) in your test data so the csv generated has empty entires.

Actually, `lastName` (and `comments` (irrelevant to this)) should not be `@Nullable`.

Reply:
Some of the students in the test data have a value for `lastName`. I was just following the `api-output` data type.

Reply:
Sure. I miss them in the first glance.

While `lastName` and `comments` should not be `@Nullable` against `master` is still a valid issue.